### ISSUE TITLE
Boots to black screen after loading perfectly 5 times

### ISSUE THREAD

**Issue by RandomAnimeGamer** [CTurt/FreeDVDBoot#296]
My PS2:
Fat SCPH-39001
Browser - 1.20
CD Player - 1.20
PlayStation Driver - 1.10
DVD Player - 2.12U

ISO:
Burned with CDBurnerXP at 1x speed - DVD+R
Premade ISO "Some 2.10 models and all 2.12.iso"
No customizations made whatsoever, all just prebuilt from the GitHub Repo

The first few times I booted FreeDVDBoot, it was working fine. I was even able to load up an ELF for SwapMagic a couple of times, trying to get a Japanese-region game to work on my American PS2.

On the sixth attempt to boot FreeDVDBoot (would have been my third attempt at booting the SwapMagic ELF) I booted to a black screen. I waited 10 minutes before restarting my PS2, to no avail. I did this a few more times in different ways (booting with and without the disc already there, selecting disc from browser, etc.)

Just recently I made another attempt on the same disc, with no luck (it's been a few days), so I decided to put in a regular American PS2 disc. It booted fine and I was able to play the game. I burned a new DVD+R with the same ISO, and still get a black screen for many minutes. I attempted to wait even 15 minutes, to no luck.

Opened up my Spider Man 3-disc set just to test the DVD player functionality (I inserted Spider Man 3, don't hate) and it works perfectly fine. None of the potential causes or solutions seem to apply to my situation, is it possible the prebuilt ISO is being rejected by my PS2? Or is there some form of customization I need to make for this to work? I'm open to any ideas.

![Spider-Man 3 1](https://i.imgur.com/OxNn9X9.png)
![Spider-Man 3 2](https://i.imgur.com/JnRCipp.png)
![Spider-Man 3 3](https://i.imgur.com/m6oihnl.png)

**Comment by RandomAnimeGamer** [comment]
Attempted to disable High DVD Compatibility and enable verify data, as one other post pointed out. Still didn't work.

**Comment by RandomAnimeGamer** [comment]
Attempted to write at 4x both with and without compatibility. Still didn't work.

**Comment by RandomAnimeGamer** [comment]
Attempted on DVD-R at 1x speed, compatibility and finalize turned on, still a black screen

**Comment by RandomAnimeGamer** [comment]
Attempted on another DVD-R at 1x speed, finalized turned on, compatibility turned OFF, still a black screen

**Comment by RandomAnimeGamer** [comment]
Attempted on another DVD-R using a different ISO "3.04 only - M+maybe other regions except J - English language.iso" on 1x, compatibility and finalize on, it launched the DVD player before giving "Unable to read disc" on the DVD menu

**Comment by RandomAnimeGamer** [comment]
Attempted on another DVD-R using a different ISO "All PS2 Slims - English language.iso" on 1x, compatibility and finalize on, it launched the DVD player before giving "Unable to read disc" on the DVD menu

**Comment by RandomAnimeGamer** [comment]
Until someone else is able to help, I've given up on FreeDVDBoot and will be using Swap Magic instead.

**Comment by ghost** [comment]
39001 can't read DVD+R afaik , maybe with DVD-ROM booktype. That 1x speed in CDBurnerXp is actually fake . The burner will just burn at it's minimal speed which can be 4x or even 8x . Older burners will be able to burn the same disc at real lower speed . Imgburn will show the speed correctly . But in you case it looks like a lens problem . Try to clean it with isopropyl alcohol or even adjust the potentiometers .

**Comment by RandomAnimeGamer** [comment]
> 39001 can't read DVD+R afaik , maybe with DVD-ROM booktype.

Doesn't matter, tried DVD-R too and it didn't work after the first 5 times, even on the same disc. Please read the original post.

> That 1x speed in CDBurnerXp is actually fake . The burner will just burn at it's minimal speed which can be 4x or even 8x .

Yet it worked the first 5 times and suddenly that same one disc stopped working... Please read the original post.

> Older burners will be able to burn the same disc at real lower speed . Imgburn will show the speed correctly . But in you case it looks like a lens problem . Try to clean it with isopropyl alcohol or even adjust the potentiometers .

Not a lens problem at all. If it was a lens problem, I wouldn't be able to play other DVDs just fine, or other games entirely. This is purely a FreeDVDBoot issue.

**Comment by michealthefox** [comment]
i had the same problem with my 70001 3.10U model like it booted the crash tag team racing game once but it didn’t again after just going to black screen and it did the same with my other burned games and i put my regular ps2 games and even my dvd movies in and they work fine like idk what’s causing the problem

**Comment by RandomAnimeGamer** [comment]
I gave up on this mod tbh, since it hasn't been updated in 3 years and nothing works. Now I just use a Swap Magic disc, which works perfectly. It's pretty clear FreeDVDBoot is abandonware imo

**Comment by DjMGameZone** [comment]
I've the same problem as you, i got the same console 39001, 2.12U dvd driver, and I can't get FDVDB to run, I burnt the ISO in many ways, even I've bought couple of FMCB memory card preinstalled, with OPL, Funtuna, Etc, but at the time to reach the Ulaunchef, OPL ,FMCB to boot...it got stuck into a black screen...depending on the memory card used, the FMCB splash screen is shown but no more than that step, with the funtuna method the console boot but it got stuck into a black screen with a little square at the top left corner... another thing that could I mention is that I try to load Ulaunchef with the independence exploit cd, at the very begining, the cd load and its looks like some files were writen (dunno where) and after that the console, no matter the way i try, it got stuck into a black screen the step before to load ulaunchef... I was thinking if one of those ISO have some data which is wrote into a nand or something like that and refuse or block the console to load the Ulaunchef, OPL, FMCB (maybe sounds crazy, but I dont know why that app (Ulaunchef) works for me in the past for 2 or 3 times and then nevermore.
If some one has any update on this kind of console model, and dvd driver pls advise to try to figure out whats wrong with this...

**Comment by RandomAnimeGamer** [comment]
> I've the same problem as you, i got the same console 39001, 2.12U dvd driver, and I can't get FDVDB to run, I burnt the ISO in many ways, even I've bought couple of FMCB memory card preinstalled, with OPL, Funtuna, Etc, but at the time to reach the Ulaunchef, OPL ,FMCB to boot...it got stuck into a black screen...depending on the memory card used, the FMCB splash screen is shown but no more than that step, with the funtuna method the console boot but it got stuck into a black screen with a little square at the top left corner... another thing that could I mention is that I try to load Ulaunchef with the independence exploit cd, at the very begining, the cd load and its looks like some files were writen (dunno where) and after that the console, no matter the way i try, it got stuck into a black screen the step before to load ulaunchef... I was thinking if one of those ISO have some data which is wrote into a nand or something like that and refuse or block the console to load the Ulaunchef, OPL, FMCB (maybe sounds crazy, but I dont know why that app (Ulaunchef) works for me in the past for 2 or 3 times and then nevermore. If some one has any update on this kind of console model, and dvd driver pls advise to try to figure out whats wrong with this...

At this point I gave up, this boot method is fucking abandonware at this point, hasn't been updated in 5 years

#370

**Comment by DjMGameZone** [comment]
Yes I know...but as far as I search through the web within this years, I havent found anybody who solve this problem..in fact you are the only one i found with the same issue...

**Comment by RandomAnimeGamer** [comment]
> Yes I know...but as far as I search through the web within this years, I havent found anybody who solve this problem..in fact you are the only one i found with the same issue...

Abandonware means you probably won't get an answer. Fundamentally this method is flawed and nobody is there to improve it.
